docs/user/plagiarism-check.rst

```diff
@@ -0,0 +1,162 @@
+2. Selected submission. When the user selects the submission from the submission list on the left side of the plagiarism run result view, new actions can be selected to initiate the plagiarism check workflow. We will provide further details to the workflow in the next section.
+ |run-results-selected-submissions|
+
+Plagiarism Check Workflow
+------------------------
+
+Tutors
+^^^^^^
+The tutors are responsible for checking the plagiarism attempts of students. They execute the plagiarism checks and mark the potential plagiarism candidates to notify the instructors.
+This process is visualized in the following diagram.
+
+|tutor-workflow|
+
+1. Open the Exercise via: Course Management → Course → Exercises → Your Exercise".
+2. Access the plagiarism tool as seen in the picture
+ |exercise-page|
+
+3. Run the Plagiarism Check.
+ a. Tutor can adjust the similarity threshold and other settings as described in section :ref:`settings` if deemed necessary (most of the time not, depends on the exercise type).
+ b. Checking big exercises can take a long time (sometimes multiple hours) and be very resource heavy → Start the checks at times when Artemis is less used
+4. Start checking for plagiarism.
+ a. Assess if the presented matches are real plagiarism or not.
+ b. Depending on your assessment, either deny or confirm the match as plagiarism.
+ c. Continue until the matches start to get "too different".
+
+
+Instructors
+^^^^^^^^^^^
+The instructors can execute the same actions as tutors, additionally they are able to make a final decision on the particular confirmed plagiarism case.
+The process is visualized in the following diagram.
+
+|instructors-workflow|
+
+1. Open the plagiarism cases via: Course Management → Course → Plagiarism Cases.
+ |plagiarism-cases-navigation|
+
+2. Overview the current status of all confirmed plagiarism cases by inspecting the management overview. The overview provides the following information about each confirmed case:
+ |confirmed-plagiarism-cases-management|
+
+ a. In how many comparisons the confirmed case appears.
+ b. Student was notified or not.
+ c. Student has responded or not.
+ d. Final verdict.
+
+3. Export the confirmed cases as CSV file to analyze them in another editor.
+4. Select one confirmed case and navigate to the case detail view.
+ |selected-confirmed-case|
+
+5. Notify the student about the potential plagiarism case and ask him or her for statement.
+ |student-notification|
+
+6. Make a final verdict by selecting one of 3 available options. The final verdict must be approved by clicking ok in the dialog.
+ |verdict-dialog|
+
+ a. Point deduction in % (minimum value 0%, maximum 100). Deduct points and confirm  the verdict in the dialog.
+ b. Warning. The instructor can write a warning message to the student and confirm the verdict by in the dialog.
+ c. Plagiarism. Next steps must be made manually by contacting responsible persons in the university to mark the student grade as "Unterschleif".
+
```

Comment:
I think the above actions could use a little more explanation (how do deducted points work in regard to bonus points, what happens with the warning message).

Reply:
Deduction: Its only relevant for the current exercise
Warning: only notification that big brother is watching you